.env

```diff
@@ -4,4 +4,21 @@ VUE_APP_featureURL = /services/vendors
 
 VUE_APP_vendorURL = /vendor/featured/0/100
 
-VUE_APP_allServiceURL = /services
\ No newline at end of file
+VUE_APP_allServiceURL = /services
+
+VUE_APP_vendorReg = /vendor/new
+
+VUE_APP_verify = /identity/verify
+
+VUE_APP_login = /identity/login
+
+VUE_APP_fbLogin = /identity/register/facebook
+
+VUE_APP_cloudName = saedconnect
+
+VUE_APP_preset = totalcost
+
+VUE_APP_tags = browser-upload
```

Comment:
Can this naming follow a similar convention e.g VUE_APP_YY_XX, A mismatch of upper and lower case looks untidy

Reply:
This has been resolved.